### ISSUE TITLE
Wrong highlighting in "UNICODE_CHAR %02d:%02d" % val

### ISSUE THREAD

**Issue by averrin** [MagicStack/MagicPython#15]
![mp_glitch](https://cloud.githubusercontent.com/assets/426007/10640251/9a02a3bc-781c-11e5-9cd8-e5af6006a530.png)

**Comment by 1st1** [comment]
Thanks for reporting. Are you using the latest version of MagicPython -- v0.4.29? Also, what color theme is it? Is it ST or Atom?

**Comment by averrin** [comment]
Yeah, 0.4.29
Atom, Base16 Tomorrow Dark

**Comment by 1st1** [comment]
Can't reproduce :(

Could you please send me this snippet of code, preferably with +10 lines from above/below.  Also, what's the package name for the theme in Atom?

And lastly, could you please double check that it's MagicPython that highlights this file?  (In the bottom-right corner of the editor it should say "MagicPython")

**Comment by averrin** [comment]
```python
t, delta = getTime(ret_delta=True)
if delta.startswith('-'):
    delta = 'Go home NOW!'
else:
    delta += ' left'
print("  🏠 Home time: %s (%s)" % (t, delta))
t, delta = getTime(True, ret_delta=True)
if delta.startswith('-'):
    delta = 'Go home NOW!'
else:
    delta += ' left'
print("  🕐 Ideal home time: %s (%s)" % (t, delta))
print("  🕐 Current average: %02d:%02d" % getAverage())

try:
    route = fetchRoute()
    while not route.strip():
        route = fetchRoute()
    route = json.loads(route)
    duration = route['data']['features'][0]['properties']['RouteMetaData']['Duration']['text']
    duration = duration.encode('utf8')
    print('  🚗 Route to home: %s' % duration)
    print('')
except:
    pass
```

https://atom.io/themes/base16-tomorrow-dark-theme

Yes, MagicPython. 100%

**Comment by averrin** [comment]
Reproduces with only one line and with Atom Dark theme

**Comment by 1st1** [comment]
Now I can reproduce it

**Comment by 1st1** [comment]
@averrin This seems to be a bug in the [oniguruma](https://en.wikipedia.org/wiki/Oniguruma) regex engine that Atom uses. Sublime Text, for instance, works just fine.

At this point I doubt it's in our power to fix this, we should actually file a bug to Atom.  BTW, the standard `language-python` syntax also breaks in exactly the same way.

**Comment by averrin** [comment]
Oh, u r right, standard highlighter breaks too.

**Comment by 1st1** [comment]
I'll close this issue, since the problem is not on our end :(

**Comment by 1st1** [comment]
Looks like this has been fixed in Atom 1.1!
